lib/browserify.js

```diff
@@ -6,12 +6,7 @@ const transform = require('./js-transform')
 
 module.exports = browserifyIt
 
-function browserifyIt ({
-  config,
-  entry,
-  env,
-  minify
-}) {
+function browserifyIt({config, entry, env, minify}) {
```

Comment:
Btw, I agree in that I'd rather not change this particular rule across our code base. But I also would rather use a standard formatter much more than have spaces before function names.